soccer/gameplay/skills/intercept.py

```diff
@@ -18,7 +18,8 @@ def __init__(self, pos=None):
                             'immediately')
 
     def execute_running(self):
-        if self.robot != None and main.ball().valid:
+        #pylint: disable=no-member
```

Comment:
Pylint seems to be choking on the python properties, thinking they are methods instead of actual variables. I probably need to submit a bug report for that.